### ISSUE TITLE
Generative Design v21.6.0 for Revit 2021 causes alignment of all algorithm's nodes

### ISSUE THREAD

**Issue by jeczmien94** [DynamoDS/Dynamo#10789]
## Dynamo version
Dynamo Revit 2.5.0.7586

## Operating system
Windows 10

## What did you do? 
After installation of Generative Design v21.6.0 for Revit 2021 from here (https://manage.autodesk.com/cep/#products-services/updates), every opening of saved algorithm causes replacement of its nodes to such a form:

![2020-06-22_10h14_15](https://user-images.githubusercontent.com/48924657/85264547-24355300-b471-11ea-81f1-94818f566b31.png)

It seems, that all nodes are aligned to the left and to the top. Also, all groups are removed. It happens not only on the algorithms created and saved before installation of Generative Design, but also after re-opening of the just created algorithms. 

I'm sure that the issue is caused by Generative Design because after uninstallation everything works fine.

Because of this issue, work with algorithms is not possible. I hope it can be solved.

**Comment by jeczmien94** [comment]
It seems that it's not the only issue. I cannot run the algorithm as well and newly created code blocks cannot be edited and look like this:
![2020-06-22_11h47_35](https://user-images.githubusercontent.com/48924657/85273714-3ec1f900-b47e-11ea-9594-69175e26fedd.png)

**Comment by saintentropy** [comment]
HI @jeczmien94 Can you give us a little more information on how you are reproducing this.  You install the patch Generative Design v21.6.0 and then open any Dynamo file?

**Comment by saintentropy** [comment]
@jeczmien94 It would also help if you can show us the package you have installed

**Comment by jeczmien94** [comment]
Hello, thank You for the response.
Yes - I've installed the patch Generative Design v21.6.0, then opened Revit and Dynamo for Revit. Then I've opened the dynamo algorithm file and the result is shown on the screenshot in the first post. As I've written, the script cannot be started.

Below are my packages, that are currently installed. However, as I've written, the issue is directly caused by GenerativeDesign. 

![2020-06-22_22h11_03](https://user-images.githubusercontent.com/48924657/85330841-67c1a880-b4d5-11ea-8996-2ebf59c0c207.png)

The one crossed out with the red line is my own package, which is not published yet.

**Comment by saintentropy** [comment]
Thanks @jeczmien94!  One more question.  Can you check the contents of the C:\ProgramData\Autodesk\RVT 2021\Dynamo\2.5\packages folder to verify that there are no packages in this folder

**Comment by jeczmien94** [comment]
Yes, there is a folder called "GenerativeDesignRevit". I suppose it's been created with the installation of Generative Design v21.6.0 for Revit 2021 (modification date indicates it). Below are screenshots with the following folders and files.

![2020-06-23_22h50_45](https://user-images.githubusercontent.com/48924657/85460870-fc92d780-b5a3-11ea-9a93-dd22ebaba976.png)

![2020-06-23_22h51_19](https://user-images.githubusercontent.com/48924657/85460943-13392e80-b5a4-11ea-9b05-b7de08cdb255.png)

![2020-06-23_22h49_43](https://user-images.githubusercontent.com/48924657/85460745-dff69f80-b5a3-11ea-8eb7-600182c0e452.png)

![2020-06-23_22h49_56](https://user-images.githubusercontent.com/48924657/85460756-e258f980-b5a3-11ea-8fe2-e495855c3069.png)

**Comment by jeczmien94** [comment]
Hello,
Is there any solution to the problem?
